parser/sql/dialect/oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/type/OracleDALStatementVisitor.java

```diff
@@ -32,4 +35,21 @@ public final class OracleDALStatementVisitor extends OracleStatementVisitor impl
     public ASTNode visitAlterResourceCost(final AlterResourceCostContext ctx) {
         return new OracleAlterResourceCostStatement();
     }
+    
+    @Override
+    public ASTNode visitExplain(final ExplainContext ctx) {
+        OracleExplainStatement result = new OracleExplainStatement();
+        OracleDMLStatementVisitor visitor = new OracleDMLStatementVisitor();
+        visitor.setParameterMarkerSegments(getParameterMarkerSegments());
```

Comment:
Maybe we should merge parameterMarkerSegments in OracleDMLStatementVisitor into parameterMarkerSegments in OracleDALStatementVisitor.

```
getParameterMarkerSegments().addAll(visitor.getParameterMarkerSegments());
```

Reply:
Why should we merge？

Reply:
If the parameters are not merged, the number of parameters in the parsing result will be less than the actual value.